config/webpack/webpack.bundle.config.js

```diff
@@ -7,9 +7,9 @@ module.exports = {
   entry: helpers.root('.srctmp/index.ts'),
   output: {
     path: path.resolve(__dirname, '..', '..', 'dist', 'bundles'),
-    filename: 'core.umd.js',
+    filename: 'stache.umd.js',
```

Comment:
I used "core" so that any library can use it. Are you assuming some level of configuration, here, for builder version?